tests/snapshots.rs

```diff
@@ -439,6 +439,7 @@ fn convert_wgsl() {
             Targets::SPIRV | Targets::METAL | Targets::HLSL | Targets::WGSL | Targets::GLSL,
         ),
         ("extra", Targets::SPIRV | Targets::METAL | Targets::WGSL),
+        ("push-constants", Targets::GLSL),
```

Comment:
Since `extra` enables push constants, could we just enable it instead of the new test?

Reply:
I wrote about it in my comment, `extra.wgsl` uses `f64`s which goes doesn't support

Reply:
oh sorry, didn't see this